modules/http/http.c

```diff
@@ -24,6 +24,12 @@
 #include "http-worker.h"
 
 /* HTTPDestinationDriver */
+void
+http_dd_set_on_error_params(LogDriver *d, OnErrorParams *on_error_params)
```

Comment:
this is not just a setter, rather it inserts an entry into the on-error hash, so the name should indicate that.

Reply:
Fixed.